Robert Khromov/src/components/WelcomeModal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+export default class Welcome extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            display: true
```

Comment:
```suggestion
            display: false
```
нужно чтобы компонент `componentDidMount()` исправил `display` на `true`